### ISSUE TITLE
Fresh markets don't display history after opening for the first time

### ISSUE THREAD

**Issue by maltekliemann** [zeitgeistpm/ui#924]
This market was created quite recently and doesn't display a history: https://app.zeitgeist.pm/markets/139.

<img width="1920" alt="Screenshot 2023-03-24 at 10 34 12" src="https://user-images.githubusercontent.com/49552287/227482849-51b226e0-cf0f-410b-9ecf-edea9cac401e.png">

Refreshing the browser doesn't help, but clicking on the _Markets_ button in the menu on the left and then opening the market view again makes the history show up.

**Comment by maltekliemann** [comment]
The expected behavior is that either the market history shows up and displays at least one data point (the initial price)...

<img width="1647" alt="Screenshot 2023-04-13 at 11 39 21" src="https://user-images.githubusercontent.com/49552287/231719920-9b9c1ca9-b9cf-42f4-8522-c2fe886f604a.png">

...or, if that's not possible, the history shows up and displays some user-friendly error message, for example like this:

<img width="1920" alt="Screenshot 2023-04-13 at 11 41 48" src="https://user-images.githubusercontent.com/49552287/231720657-66dc0d5f-a80e-48d6-81e3-9bc38ecf37de.png">

**Comment by Robiquet** [comment]
> @Robiquet Is this done?

No, needs design work if we want to go with the Hydradx suggestion

**Comment by saboonikhil** [comment]
Can't we have a history with just one item (`Market opened` as per screenshot below) ?

![Screenshot 2023-11-14 at 5 06 29 PM](https://github.com/zeitgeistpm/ui/assets/36529278/14765403-a650-41b5-8709-6ec494d18f7b)

Looks like history is only visible after resolution.

![Screenshot 2023-11-14 at 5 12 18 PM](https://github.com/zeitgeistpm/ui/assets/36529278/adb835d3-d91e-4621-8a91-81dd8f9e84e1)
